_includes/content.html

```diff
@@ -94,28 +105,22 @@
 {% highlight ruby %}
 Griddler.configure do |config|
   config.processor_class = EmailProcessor # MyEmailProcessor
-  config.to = :token # :full, :email, :hash
-  # :raw    => 'AppName <s13.6b2d13dc6a1d33db7644@mail.myapp.com>'
-  # :email  => 's13.6b2d13dc6a1d33db7644@mail.myapp.com'
-  # :token  => 's13.6b2d13dc6a1d33db7644'
-  # :hash   => { raw: '', email: '', token: '', host: '' }
+  config.processor_method = :process
   config.reply_delimiter = '-- REPLY ABOVE THIS LINE --'
   config.email_service = :sendgrid
 end
 {% endhighlight %}
 
 <ul>
-<li>
-<code>config.processor_class</code> is the class Griddler will use to handle your incoming emails.</li>
-<li>
```

Comment:
A rogue LI shows it's head! 

![screen shot 2015-03-05 at 10 14 54 am](https://cloud.githubusercontent.com/assets/463175/6507528/820d1776-c320-11e4-9b97-3ebe9003926a.png)